### ISSUE TITLE
Where do you put your keywords/commands?

### ISSUE THREAD

**Issue by Mouvedia** [conventional-commits/conventionalcommits.org#177]
I am talking about stuff like "smart commits".
e.g. https://help.github.com/en/articles/closing-issues-using-keywords

**Comment by epage** [comment]
If I'm reading your question correctly, it seems like this generally falls under the footer

> A footer of one or more lines MAY be provided one blank line after the body. The footer MUST contain meta-information about the commit, e.g., related pull-requests, reviewers, breaking changes, with one piece of meta-information per-line.

And the frontpage even includes an example

```
fix: correct minor typos in code

see the issue for details on the typos fixed

closes issue #12
```

**Comment by Mouvedia** [comment]
It was more of a poll but yeah I guess this is already covered by the seventh point of the specification.
